docs/DotConfig.md

```diff
@@ -2,20 +2,20 @@ Mock Firebolt: mf.config.json(Supported SDK's)
 =======================
 ## Supported SDK's
 
-Mock Firebolt supports these Firebolt SDKs: core, manage, discovery.
-mf.config.json file contains a list of SDK's supported by Mock Firebolt, Out of which `core` is mandatory and other two `manage` and `discovery` are optional.
+Mock Firebolt supports these Firebolt SDKs: core, discovery.
+mf.config.json file contains a list of SDK's supported by Mock Firebolt, Out of which `core` is mandatory and `discovery` is optional.
```

Comment:
We should probably talk about this. The SDK-ness of this feature is hard to support now that the open rpc for both core and manage (and maybe discovery in the future) is all in one OpenRPC file.